lib/scope.js

```diff
@@ -1040,12 +882,11 @@ module.exports = {
     */
     // There should only ever be one match for any table row and for any field row.
     // Not sure how to check on this.
-    let matches = [];
+    let matches_by_field = {};
```

Comment:
This will be an object where each field has a key and its value will be an array of matching table rows.

Reply:
Comment made.